languagetool-language-modules/pt/src/main/java/org/languagetool/tokenizers/pt/PortugueseWordTokenizer.java

```diff
@@ -162,34 +162,32 @@ public List<String> tokenize(final String text) {
   /* Splits a word containing hyphen(-) if it doesn't exist in the dictionary. */
   private List<String> wordsToAdd(String s) {
     final List<String> l = new ArrayList<>();
-    synchronized (this) { // speller is not thread-safe
```

Comment:
can we remove the synchronized in `EnglishWordTokenizer` and `SpanishWordTokenizer` as well?

Reply:
I'll take a look; those didn't show up in the profiling data